test/spec/differ.js

```diff
@@ -292,7 +309,7 @@ describe('diffing', function() {
 
         expect(results._changed).to.have.keys([
           '_6-127',
-          '_6-219'
+          '_6-220'
```

Comment:
Based on my understanding, since I added a new id nested under `_6-220` which is the actual element being changed, this is the correct behaviour.